src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -105,6 +97,20 @@ public static void merge(List<RepoConfiguration> repoConfigs, List<AuthorConfigu
         }
     }
 
+    private static RepoConfiguration getMatchingRepoConfig(
+            List<RepoConfiguration> repoConfigs, AuthorConfiguration authorConfig) {
+        RepoConfiguration matchingRepoConfig = null;
+
+        for (RepoConfiguration repoConfig: repoConfigs) {
+            if (repoConfig.getLocation().equals(authorConfig.getLocation())
+                    && repoConfig.getBranch().equals(authorConfig.getBranch())) {
+                matchingRepoConfig = repoConfig;
```

Comment:
can just return `repoConfig` to end the method, no need for break.

Reply:
Oops sorry, my bad. Made the change.